ArduCopter/mode_sport.cpp

```diff
@@ -116,6 +116,9 @@ void ModeSport::run()
         break;
     }
 
+    // set roll/pitch rate shaping time constants
+    attitude_control->set_roll_pitch_rate_tc(g2.acro_rate_rp_tc);
+
```

Comment:
@rmackay9 Why does sport mode use the pilot yaw rate parameter but then uses acro pitch and roll rate parameter?

Reply:
@bnsgeyer, Sport mode is compiled out by default in 4.2 and I'll remove it for 4.3 so let's just be sure that we don't break it completely (for 4.2).

Reply:
It probably should have used all the acro parameters but it will be gone soon.